bberg/src/flavor_builder.rs

```diff
@@ -35,7 +38,15 @@ impl FlavorBuilder for BBFiles {
         all_cols_and_shifts: &[String],
     ) {
         // TODO: move elsewhere and rename
```

Comment:
Wouldn't it be the time to address or remove this TODO?

Reply:
will address